### PR TITLE
Portal [izlude to lasagna, lasagna to izlude ,lasagna to malangdo  ]

### PR DESCRIPTION
**Pull request by sctnightcore**
Need test !

### CODE REVIEW

File: tables/portals.txt [hunk 1]
```diff
@@ -3149,6 +3149,9 @@ dew_fild01 48 65 dew_dun02 302 30
 dew_dun02 305 30 dew_fild01 50 65
 
 # Lasagna
+izlude 198 213 lasagna 206 297 2000 c c r0 n
```
Comment (Fadreus) [inline tables/portals.txt]: I updated a couple of portals on rock ridge yesterday along with lasagna.
For iRO from izlude to lasagna is:

izlude 195 213 lasagna 206 297 2000 c c r0 n

Comment (Fadreus) [inline tables/portals.txt]: I updated a couple of portals on rock ridge yesterday along with lasagna.
For iRO from izlude to lasagna is:

izlude 195 213 lasagna 206 297 2000 c c r0 n

Comment (Fadreus) [reply tables/portals.txt]: 198 213 is to Pharos Lighthouse (for iRO)

Comment (sctnightcore) [reply tables/portals.txt]: oh sry

**Pull request merged**